docs/grid_model/network_subnetwork.md

```diff
@@ -819,9 +820,38 @@ A busbar section is a non impedant element used in a node/breaker substation top
 (internal-connection)=
 ## Internal connection
 
-**Internal connection**  
+[![Javadoc](https://img.shields.io/badge/-javadoc-blue.svg)](https://javadoc.io/doc/com.powsybl/powsybl-core/latest/com/powsybl/iidm/network/VoltageLevel.NodeBreakerView.InternalConnection.html)<br>
 An internal connection is a non-impedant connection between two components in a voltage level.
 
 <!--
 <span style="color:red"> TODO</span>
--->
\ No newline at end of file
+-->
+
+(overload-management-system)=
+## Overload management systems
+
+[![Javadoc](https://img.shields.io/badge/-javadoc-blue.svg)](https://javadoc.io/doc/com.powsybl/powsybl-core/latest/com/powsybl/iidm/network/OverloadManagementSystem.html)<br>
+An overload management system is an automation system that makes network actions when a current limit condition is met.
```

Comment:
An overload management system is an automation system that monitors the current on a side of a network element. It can be enabled or not. The system contains a list of trippings. A tripping here is an action to apply if a limit is reached in the monitored location. The supported actions are...